src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt

```diff
@@ -42,7 +42,7 @@ enum class InmateRepositorySql(val sql: String) {
         O.ROOT_OFFENDER_ID,
         O.OFFENDER_ID_DISPLAY OFFENDER_NO,
         O.FIRST_NAME,
-        CONCAT(O.middle_name, CASE WHEN middle_name_2 IS NOT NULL THEN concat(' ', O.middle_name_2) ELSE '' END) MIDDLE_NAME,
+        trim(concat(trim(O.middle_name), concat(' ', trim(O.middle_name_2)))) MIDDLE_NAME,
```

Comment:
Tested against real NOMIS data and this does indeed trim everything properly with any combination of middle names.

Reply:
what about where only `middle_name_2` is supplied?